### PR TITLE
Fix NPE in CheckListView 

### PR DESCRIPTION
**Pull request by samhutchins**
As requested, a PR with a fix for #1219

### CODE REVIEW

**Comment by abhinayagarwal** [comment]
Hi @samhutchins ,

Thanks for the PR, have you signed the [CLA](https://docs.google.com/forms/d/e/1FAIpQLSfGmJDMdl9mPBiZLwhUXPNRKcSSJDawBHe_ED2ReSJGUt-puQ/viewform) as mentioned in the [Contributing doc](https://github.com/controlsfx/controlsfx/wiki/Contributing-to-ControlsFX)?

**Comment by samhutchins** [comment]
I did, yeah. myname at screamingfrog

File: controlsfx/src/main/java/org/controlsfx/control/CheckListView.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2013, 2018 ControlsFX
+ * Copyright (c) 2013, 2018, 2019 ControlsFX
```
Comment (abhinayagarwal) [inline controlsfx/src/main/java/org/controlsfx/control/CheckListView.java]: This should be Copyright (c) 2013, 2019 ControlsFX

Comment (samhutchins) [reply controlsfx/src/main/java/org/controlsfx/control/CheckListView.java]: Apologies, I mis-interpreted the contributing guide. I've pushed a second commit with the requested change

**Pull request merged**